NEWS

```diff
@@ -1,3 +1,52 @@
+KLEE 1.4.0, 20 July 2017
+========================
+(Incorporating changes from 4 November 2016 up to and including 20 July 2017)
+
```

Comment:
It would be good to see a note here stating that

* This is last release that will support LLVM 2.9
* This is the last release that will support the Autoconf/Makefile build system

Users should prepare for these to be removed in the future.